scripts/generate_insolar_configs.go

```diff
@@ -150,6 +150,19 @@ func main() {
 			conf.Ledger.JetSplit.DepthLimit = 4
 		}
 
+		// if node.Role == "heavy_material" {
```

Comment:
Add comment to this section, why its here

Reply:
Why is it commented?